recipes/mingw-w64/linux/conanfile.py

```diff
@@ -0,0 +1,324 @@
+import os
+from conans import ConanFile, tools, AutoToolsBuildEnvironment
+from conans.errors import ConanInvalidConfiguration
+
+
+class MingwConan(ConanFile):
+    name = "mingw-w64"
+    description = "MinGW is a contraction of Minimalist GNU for Windows"
```

Comment:
Config and validate say linux 😖 


```suggestion
    description = "MinGW is a contraction of Minimalist GNU for Linux"
```

Reply:
You want to invent MinGL ? I think you might be on something here !